bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/impl/GemsStrategy.scala

```diff
@@ -27,7 +27,10 @@ import edu.gemini.spModel.core.{Angle, MagnitudeBand}
 import scalaz._
 import Scalaz._
 
-object GemsStrategy extends AgsStrategy {
+trait GemsStrategy extends AgsStrategy {
+  // By default use the remote backend but it can be overriden in tests
+  private [impl] implicit val backend:VoTableBackend = RemoteBackend
```

Comment:
I guess this use of implicit params does seem a bit magical.  If it is too awkward to add the backend option everywhere `GemsVoTableCatalog.search` is called, it seems like you could accomplish the same thing in other ways.  Instead of the `GemsStrategy` being fitted with an implicit value for the backend, `GemsVoTableCatalog` itself could be parameterized by the backend and then a different instance used for testing.  I don't know if that helps.